src/components/CartCounter/styled.js

```diff
@@ -1,8 +1,22 @@
-import styled from 'styled-components';
+import styled, {css} from 'styled-components';
 const StyledCartCounter = styled.span`
 	margin: 1rem;
 	padding: 0.6rem;
 	color: whitesmoke;
 	font-weight: bold;
+	${({variant}) =>
+		variant === 'counterHome' &&
+		css`
+			display: inline-block;
+			margin: 0rem;
+			padding: 0rem;
```

Comment:
I'm not 100% sure, but I guess these properties have no effects.

Anyway if you set a value to  `0` there is no need to set a unit 🧐
```suggestion
			margin: 0;
			padding: 0;
```